lib/pages/home/survey_carousel/survey_carousel_card.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:survey/pages/home/survey_carousel/survey_ui_model.dart';
+
+// ignore: must_be_immutable
+class SurveyCarouselCard extends StatelessWidget {
+  SurveyCarouselCard({this.surveyUiModel});
+
+  final SurveyUiModel surveyUiModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
```

Comment:
by default `Container` widget will take as much as possible size, so we don't need define max size here 🙆‍♂️